test/common/testlib.py

```diff
@@ -784,9 +788,6 @@ def skipImage(reason, *args):
     return lambda func: func
 
 class Policy(object):
-    def __init__(self):
-        self.retryable = True
```

Comment:
I was thinking that we have one TapResult per test, but it's the same as with Policy.  Still, it feels more natural to have the retry state in the TapResult, where all the other state is as well.